demos/demo_runner/iot_demo_runner.c

```diff
@@ -1,6 +1,6 @@
 /*
- * Amazon FreeRTOS V201912.00
- * Copyright (C) 2019 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ * FreeRTOS V201912.00
```

Comment:
Shouldn't the version number get changed as well?

Reply:
version number is supposed to be updated by another tool.

Reply:
Ah  I see.

Reply:
I think if we are going to change every file in the repo it is better to do it once than to do it twice. Lets update the version number as well in this repo. It will never be possible to separate these 2 changes as they all affect the same lines ...